src/fsharp/FSharp.Core/array.fsi

```diff
@@ -12,6 +12,14 @@ namespace Microsoft.FSharp.Collections
     [<RequireQualifiedAccess>]
     module Array = 
 
+        /// <summary>Builds a new array that contains the cartesian product of the two input arrays.</summary>
```

Comment:
Please use a less technical term than "Cartesian product" in the comment, such as "contains all pairings of elements from the first and second arrays", likewise the other three signature file comments